sources/common/source_utils.go

```diff
@@ -0,0 +1,18 @@
+package common
+
```

Comment:
In general we have a test file for every go file.
Consider adding a utils_test.go for testing ToNotNull?